doc/datasets/index.rst

```diff
@@ -351,6 +351,8 @@ features::
 
  _`Faster API-compatible implementation`: https://github.com/mblondel/svmlight-loader
 
+.. include:: openml.rst
```

Comment:
The sections seems strange:
```
5.6. Downloading datasets from the openml.org repository
5.6.1. Dataset Versions
5.6.2. Loading from external datasets
```
Maybe put it at the end of the file? (or somewhere else)

Reply:
It's an issue with header levels :-/